uaclient/cli/__init__.py

```diff
@@ -27,7 +27,7 @@
     lock,
 )
 from uaclient import log as pro_log
-from uaclient import messages, security_status
+from uaclient import messages, secretManager, security_status
```

Comment:
Naming conventions recommend that module names are defined in a single word (I don't have a better suggestion than this one so I'm fine) and, having multiple words, use snake_case.
`secret_manager`?

Reply:
Agreed, have pushed the changes.